phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexRebuildRegionScanner.java

```diff
@@ -187,13 +509,56 @@ public RegionInfo getRegionInfo() {
     @Override
     public boolean isFilterDone() { return hasMore; }
 
+    private void logToIndexToolResultTable() throws IOException {
+        long scanMaxTs = scan.getTimeRange().getMax();
+        byte[] keyPrefix = Bytes.toBytes(Long.toString(scanMaxTs));
+        byte[] regionName = Bytes.toBytes(region.getRegionInfo().getRegionNameAsString());
+        byte[] rowKey = new byte[keyPrefix.length + regionName.length];
+        Bytes.putBytes(rowKey, 0, keyPrefix, 0, keyPrefix.length);
```

Comment:
Comment here that PK is scanMaxTs + regionName would be helpful. Or alternately, extract key building into a helper method.

Reply:
I will add the comment you suggested